src/nvim/buffer.h

```diff
@@ -4,6 +4,7 @@
 #include "nvim/window.h"
 #include "nvim/pos.h"  // for linenr_T
 #include "nvim/ex_cmds_defs.h"  // for exarg_T
+#include "nvim/screen.h"  // for StlClickRecord
```

Comment:
This *is* needed by buffer.h because buffer.h.generated.h is considered to be a part of it.